### ISSUE TITLE
revive: cut output from exported rule

### ISSUE THREAD

**Issue by gucio321** [golangci/golangci-lint#1993]
**Is your feature request related to a problem? Please describe.**
Hi there, sometimes, output from revive linter is really long, like
```
exported: exported var DataVar should have comment or be unexported (revive)
var DataVar map[int]string {
1: "s1",
2:"s2",
3:"s3",
```
e.t.c.
```
2000:"s2000",
}
```
in my opinion it is a bit hard to read. is it possible to cut it?

**Describe the solution you'd like**
e.g., linter should print up to `x` lines of cod.

**Comment by ldez** [comment]
Hello,

could you provide a reproducible example?

**Comment by ldez** [comment]
Do you have other examples?

I don't think that cut the source code output is a good idea, but if it's only produced by the `exported` rule I have a solution.

**Comment by gucio321** [comment]
Hi @ldez !
thanks for really fast answer!
Yes, I think, that it is produced by `exported` only.

**Comment by gucio321** [comment]
<details><summary>example (code)</summary>

```golang
package somepackage

var DataVar = map[int]string{
        1:  "s1",
        2:  "s1",
        3:  "s1",
        4:  "s1",
        5:  "s1",
        6:  "s1",
        7:  "s1",
        8:  "s1",
        9:  "s1",
        10: "s1",
        11: "s1",
        12: "s1",
        13: "s1",
        14: "s1",
        15: "s1",
        16: "s1",
        17: "s1",
        18: "s1",
        19: "s1",
        20: "s1",
        21: "s1",
        22: "s1",
        23: "s1",
        24: "s1",
        25: "s1",
        26: "s1",
        27: "s1",
        28: "s1",
        29: "s1",
        30: "s1",
        31: "s1",
        32: "s1",
        33: "s1",
        34: "s1",
        35: "s1",
        36: "s1",
        37: "s1",
        38: "s1",
        39: "s1",
}
```

</details>

<details><summary>example (.golangci.yaml)</summary>

```yaml
linters:
  disable-all: true
  enable:
    - revive

```

</details>

**Comment by ldez** [comment]
@gucio321 #1994 :wink:
